py-usfm-parser/tests/test_json_conversion.py

```diff
@@ -116,3 +116,54 @@ def test_output_is_valid_usj(file_path):
     assert not test_parser.errors, test_parser.errors
     usj_dict = test_parser.to_usj()
     validate(instance=usj_dict, schema=USJ_SCHEMA)
+
+def remove_newlines_in_text(usj_dict):
+    '''The test samples in testsuite do not preserve new lines in. But we do in usfm-grammar.
```

Comment:
We should talk about this sometime. Okay for now.

Reply:
In our usfm-grammar way, we preserve the line breaks and multiple space as it is in input USFM. But the USXs in testsuite do not do that. Hence the USJs generated from those USXs don't have those spaces. I didn't want to remove spaces when we generate USJs. Also thought not to raise this as a problem with testsuite. Hence I handled them like this on the tests to get them pass. That s what's happening here. But we can talk about it.